Tests/SwiftSyntaxMacroExpansionTest/AccessorMacroTests.swift

```diff
@@ -44,6 +44,95 @@ fileprivate struct ConstantOneGetter: AccessorMacro {
 final class AccessorMacroTests: XCTestCase {
   private let indentationWidth: Trivia = .spaces(2)
 
+  func testAccessorOnVariableDeclWithTrailingLineCommentAndNoAccessorBlock() {
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int // hello
+      """,
+      expandedSource: """
+        var x: Int /* hello*/ {
+          get {
+            return 1
+          }
+        }
+        """,
+      macros: ["constantOne": ConstantOneGetter.self],
+      indentationWidth: indentationWidth
+    )
+
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int /// hello
+      """,
+      expandedSource: """
+        var x: Int /** hello*/ {
+          get {
+            return 1
+          }
+        }
+        """,
+      macros: ["constantOne": ConstantOneGetter.self],
+      indentationWidth: indentationWidth
+    )
+
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int = 1 /// hello
+      """,
+      expandedSource: """
+        var x: Int /** hello*/ {
+          get {
+            return 1
+          }
+        }
+        """,
+      macros: ["constantOne": ConstantOneGetter.self],
+      indentationWidth: indentationWidth
+    )
+
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x /// hello
+      """,
+      expandedSource: """
+        var x /** hello*/ {
+          get {
+            return 1
+          }
+        }
+        """,
+      macros: ["constantOne": ConstantOneGetter.self],
+      indentationWidth: indentationWidth
+    )
+  }
+
+  func testAccessorOnVariableDeclWithTrailingLineCommentAndAccessorBlock() {
+    assertMacroExpansion(
+      """
+      @constantOne
+      var x: Int { // hello
+        1
+      }
+      """,
+      expandedSource: """
+        var x: Int { // hello
+        get {
+            1
+        }
+          get {
+            return 1
+          }
+        }
+        """,
```

Comment:
I discovered that adding a trailing comment to the left brace of the accessor block will mess up the indentation levels of the generated getter.